### PR TITLE
#803 Update RiskLog.tsx

### PR DESCRIPTION
**Pull request by superhvarn**
## Changes

Removed userId property from handleCheck() method

## Notes

edited src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #803

### CODE REVIEW

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx [hunk 1]
```diff
@@ -179,7 +178,6 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
                     checked={risk.isResolved}
                     data-testid={`testCheckbox${idx}`}
                     onChange={() => handleCheck(risk)}
-                    disabled={isLoading}
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:182]: Was this line supposed to be deleted?

Comment (superhvarn) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:182]: Line 182 was not supposed to be deleted. My bad.

Comment (ryanhowe28) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:182]: all good just add it back

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
   const history = useHistory();
   const auth = useAuth();
   const { mutateAsync: createMutateAsync } = useCreateSingleRisk();
-  const { isLoading, mutateAsync: editMutateAsync } = useEditSingleRisk();
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:51]: is isLoading an unused import?

Comment (superhvarn) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx]: It is used with my latest commit.

File: src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx [hunk 3]
```diff
@@ -1,8 +1,3 @@
-/*
- * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
- * See the LICENSE file in the repository root folder for details.
- */
-
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx:5]: Can you put this comment back as well?

Comment (superhvarn) [reply src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx]: Done.

**Pull request closed without merging**